dockerfiles/openwhisk/java/Dockerfile.function

```diff
@@ -0,0 +1,8 @@
+ARG BASE_IMAGE
```

Comment:
Dockerfile is very good :-) But we should also be able to deploy the code as zip package; particularly for cloud platforms that do not support containers (Azure, GCP). Can you add that or should I take a look into that?